sources/packages/web/src/components/students/StudentAppealSharedForm.vue

```diff
@@ -0,0 +1,222 @@
+          <v-radio :value="AppealTypes.Application" class="mt-2">
+            <template #label>
+              <div class="ml-1">
+                <strong>Application</strong>
+                <p>
+                  Application specific appeals that will only impact a single
+                  application like income changes, room and board, etc.
+                </p>
+              </div>
+            </template>
+          </v-radio>
+          <v-radio :value="AppealTypes.Other">
+            <template #label>
+              <div class="ml-1">
+                <strong>Other</strong>
+                <p>
+                  Appeals that are student related (restriction or status) that
+                  don't apply to a single application.
+                </p>
+              </div>
+            </template>
+          </v-radio>
+        </v-radio-group>
+        <v-select
+          v-if="selectedAppealType === AppealTypes.Application"
+          hide-details="auto"
+          label="Application Number"
+          density="compact"
+          :items="eligibleApplications"
+          v-model="selectedApplicationId"
+          :loading="loadingEligibleApplications"
+          item-value="id"
+          item-title="applicationNumber"
+          variant="outlined"
+          class="mb-4"
+          no-data-text="No eligible applications available"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Application number')]"
+        />
+        <v-select
+          v-if="selectedAppealType === AppealTypes.Application"
+          hide-details="auto"
+          label="Appeal(s)"
+          density="compact"
+          multiple
+          :items="applicationAppeals"
+          item-title="description"
+          item-value="formName"
+          v-model="selectedApplicationAppeals"
+          variant="outlined"
+          class="mb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Appeal(s)')]"
+        />
+        <v-select
+          v-if="
+            selectedAppealType === AppealTypes.Other && isOtherOptionAvailable
+          "
+          hide-details="auto"
+          label="Appeal form"
+          density="compact"
+          :items="otherAppeals"
+          item-title="description"
+          item-value="formName"
+          v-model="selectedOtherAppeal"
+          variant="outlined"
+          class="mb-4"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Appeal')]"
+        />
+        <banner
+          v-if="
+            selectedAppealType === AppealTypes.Other && !isOtherOptionAvailable
+          "
+          class="mb-2"
+          type="info"
+          header="Other appeal types will be available for submission soon."
+        />
+      </v-form>
+    </content-group>
+    <footer-buttons
+      class="mt-4"
+      primary-label="Next"
+      justify="end"
+      @primary-click="goToAppealFormsRequests"
+      :show-secondary-button="false"
+      :show-primary-button="selectedAppealType === AppealTypes.Application"
+    />
```

Comment:
Not a blocker. For a process with next and previous steps, we can consider the stepper component if that could be more suitable. 

https://vuetifyjs.com/en/components/steppers/